src/puppetlabs/trapperkeeper/services/status/status_debug_logging.clj

```diff
@@ -0,0 +1,11 @@
+(ns puppetlabs.trapperkeeper.services.status.status-debug-logging
+  (:require [clojure.tools.logging :as log]
+            [schema.utils :refer [validation-error-explain]]
+            [cheshire.core :as json]
+            [puppetlabs.trapperkeeper.services.status.status-core :as status-core]))
+
+(defn log-status
+  "Log status information at the debug level as json"
+  []
+  (let [status (status-core/v1-status :debug)]
```

Comment:
I like your `status-latest-version` idea because it's obvious that it should be updated. `status-map-for-version` might be a good idea regardless, but it seems like you'd still have the problem of forgetting to update this function to pass `2` to `status-map-for-version`

I'll go with `status-latest-version` for now I think?